### ISSUE TITLE
Target trajectory format investigation

### ISSUE THREAD

**Issue by Locke0** [ManifoldRG/software-control#134]
Considering target agents - GTA1 and UI-TARS1.5, what are their expected data formats and trajectory formats?

What are the target dataset formats (trajectory data schemas, CoT formats, action space schemas, label formats)

How to convert the existing dataset formats to target (success & failure trajectories)?

Can we use the negative data samples from the existing datasets considering UI-TARS-1.5 (uses VeOmni codebase for training infra) and GTA1’s training scripts?

Refer to their codebases, specifically eval scripts & finetuning scripts.

GTA1:
training: https://github.com/Yan98/GTA1
eval: https://github.com/xlang-ai/OSWorld/blob/main/mm_agents/gta1_agent.py

UI-TARS:
training: https://github.com/ByteDance-Seed/VeOmni/tree/main (the original authors said this is the training infra they used)
eval: https://github.com/xlang-ai/OSWorld/blob/main/mm_agents/uitars15_v2.py

Btw, in the OSWorld eval scripts, these agents usually use vLLM for inference and openai library as client server.

In order to figure out the target trajectory formats to generate and what adaptation logic is needed, here are the deliverables:
- [ ] specific agent training data ingestion schemas
- [ ] preprocessing steps the training scripts use on trajectories

**Comment by Locke0** [comment]
Btw after some reading and reflecting, I believe we need to use offline RL methods to finetune with our dataset instead of SFT (it is prone to memorization and our dataset won't be a clean positive set of demonstrations). Luckily, GTA1 uses RL and UI-TARS also has RL for finetuning
